### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scale' of undefined

### ISSUE THREAD

**Issue by corporatepiyush** [c3js/c3#1164]
i just downloaded new version started using example on http://c3js.org/samples/simple_multiple.html

i got the error
Uncaught TypeError: Cannot read property 'scale' of undefined

**Comment by dublinus** [comment]
I get the same error. On safari, it says `TypeError: undefined is not an object (evaluating 'd3.scale')`. Is there some new requirement for the newer version?

**Comment by dublinus** [comment]
For me, this was an issue with my include of d3.js. It wasn't recognizing some of the variable names because some of them were Greek letters. If you just include the minified version (d3.min.js) before c3.js, it should work.

**Comment by rampart81** [comment]
I am getting the same error. 
I included the d3.min.js before c3.min.js but still see the error.

**Comment by rquiroga** [comment]
Thanks, I included D3 before C3 and worked.

**Comment by FoxRanger** [comment]
I have this same error ....

**Comment by delucas** [comment]
Shouldn't this be closed?

**Comment by medington** [comment]
Ran into this same error attempting to use the current non-minified versions of C3 (0.4.11) and D3 v3 (3.5.17).

When I switched to the minified versions the same simple test worked. I'm using the example from the [getting started guide](http://c3js.org/gettingstarted.html) almost verbatim.

A find it more than a little concerning that the behavior would be be different between the minified and non-minified versions. Can someone from the C3 dev team explain this for me? I'm running on Mac and using files directly downloaded from the RAW links on GitHub.

Sure feel to me like there is something here that should be fixed or an error message improved.
